device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsCbsReceiverLinkHandler.java

```diff
@@ -36,7 +36,6 @@
     AmqpsCbsReceiverLinkHandler(Receiver receiver, AmqpsLinkStateCallback amqpsLinkStateCallback)
     {
         super(receiver, amqpsLinkStateCallback, UUID.randomUUID().toString());
-        this.receiverLinkTag = RECEIVER_LINK_TAG_PREFIX;
```

Comment:
Removed this one (and other similar fields) because I am getting a warning that these links are assigned, but never really accessed. It would be great if someone in the team can confirm the validity of my action.

Reply:
Seems like it is safe to remove. As long as the AMQP e2e tests don't blow up you should be good here.

Reply:
does azp run run e2e tests?